src/app/shared/components/student-profile-apprenticeship-company/student-profile-apprenticeship-company.component.html

```diff
@@ -1,6 +1,9 @@
 <address>
-  <ng-container *ngIf="company.apprenticeshipContract.CompanyName"
-    >{{ company.apprenticeshipContract.CompanyName }}<br
+  <ng-container *ngIf="company.apprenticeshipManager?.CompanyName"
+    >{{ company.apprenticeshipManager?.CompanyName }}<br
+  /></ng-container>
+  <ng-container *ngIf="company.apprenticeshipManager?.CompanyNameAddition"
+    >{{ company.apprenticeshipManager?.CompanyNameAddition }}<br
```

Comment:
Gibt es hier einen zusätzlichen Abstand, wegen dem `<br>`, wenn nur eines der beiden vornhanden ist, oder ist das kein Problem? Vielleicht könntest du die einzelnen Zeilen einfach in ein `<div>` wrappen, welches ein Blockelement ist: `<div *ngIf="company.apprenticeshipManager?.CompanyName"
    >{{ company.apprenticeshipContract.CompanyName }}</div>`. Das wäre wohl grundsätzlich im ganzen File der Fall.

Reply:
Sollte nicht, die <br> sind ja innerhalb vom Container. Oder verstehe ich etwas falsch?
Im konkreten Beispiel, wo `CompanyNameAddition` nicht gesetzt ist, gibt es keinen Zusätzlichen Abstand.

Ansonsten müsste man es dann gleich in allen Dateien anpassen, wo wir das bisher so machen...

Reply:
Nein, ich denke es ist OK so :+1: